README.md

```diff
@@ -1,4 +1,4 @@
-# LoopBack Boot
+# LoopBack Boot 
```

Comment:
Thanks @bajtos .  Was trying to force CI to pick up new changes in `gateway-director-testapp` to get it to pass.  Will get you review this PR again when it's ready.